apps/fyle/actions.py

```
@@ -132,8 +132,8 @@ def update_failed_expenses(failed_expenses: list[Expense], is_mapping_error: boo
         error_type = 'MAPPING' if is_mapping_error else 'ACCOUNTING_INTEGRATION_ERROR'
 
         # Skip dummy updates (if it is already in error state with the same error type)
-        if not (expense.accounting_export_summary.get('state') == 'ERROR' and \
-            expense.accounting_export_summary.get('error_type') == error_type):
+        if (expense.accounting_export_summary.get('state') not in ['ERROR', 'DELETED'] and \
+            expense.accounting_export_summary.get('error_type') != error_type):
```

Flake8 (7.2.0)

[error] visually indented line with same indent as next logical line

(E129)